src/composable/chat/useChatMessages.ts

```diff
@@ -250,6 +250,21 @@ export function useChatMessages(ctx: ChannelContext) {
 		return Object.values(umap);
 	}
 
+	/**
+	 * Returns a message by the specified message id
+	 *
+	 * @param id the message id
+	 * @returns message
+	 */
+	function messageById(id: string): ChatMessage | void {
```

Comment:
This method can be removed, as in almost any case where it might be useful, the existing `awaitMessage` would cover it.